### PR TITLE
 Add poweron architecture ppc64le to travis build

### PR DESCRIPTION
**Pull request by asellappen**
This is part of the Ubuntu distribution for ppc64le. This helps us simplify testing later when distributions are re-building and re-releasing, For more info please tag @gerrith3.

### CODE REVIEW

**Comment by rhatdan** [comment]
Please sign your commit.

git commit -a --amend -s
git push --force

**Comment by asellappen** [comment]
> Please sign your commit.
> 
> git commit -a --amend -s
> git push --force

This has been done now.

**Comment by rhatdan** [comment]
LGTM
@kolyshkin @thaJeztah @mrunalp @vrothberg PTAL

File: .travis.yml [hunk 1]
```diff
@@ -25,5 +25,27 @@ jobs:
...
+    - stage: test
+      script: make test BUILDTAGS=""
+      arch: ppc64le
+      go: 1.15.x
+    - stage: test
+      script: make test BUILDTAGS=""
+      arch: ppc64le
+      go : tip
+    - stage: validate
+      arch: ppc64le
```
Comment (vrothberg) [inline .travis.yml:48]: I can think we can simplify it more.

The go versions are already specified at the top and Travis will construct the matrix on it's own. So we don't need to specify them here.

In fact, we don't need to add new stages at all but extend the existing ones with:
```
arch:
  - amd64
  - ppc64le
```

Comment (asellappen) [reply .travis.yml]: @vrothberg  , I have tried earlier and it did not work , This was the reason i have specified cpu arch under matrix .

Again i tried as you suggested and ran the jobs and only new arch included in the test stage , I am expecting it should be  included in other stages Validate,build ,lint etc.. , Please find details below 


https://travis-ci.com/github/asellappen/selinux/builds/190412878

Please let me if you have any other thoughts.

Comment (asellappen) [reply .travis.yml]: shall go for the approach which i used in matrix?

File: .travis.yml [hunk 2]
```diff
@@ -25,5 +27,6 @@ jobs:
       script: make test BUILDTAGS=""
 
     - stage: validate
-      script: |
+    
+script: |
```
Comment (thaJeztah) [inline .travis.yml:31]: Was this intentional? If I see correctly, this moves the actual script out of the validate step?

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (asellappen) [reply .travis.yml]: sorry ,it was my mistake , i have correct this move and ran the job, without matrix poweron host is not running for stages build,Validate,Tint. only Test stage is included  power on arch as per below run. 

refer https://travis-ci.com/github/asellappen/selinux for details.

This was the reason i used matrix to run package for both cpu arch.

please check and pass your inputs.

**Comment by asellappen** [comment]
https://github.com/opencontainers/selinux/pull/121 created newly

**Pull request closed without merging**